docs/hub/datasets-upload-guide-llm.md

```diff
@@ -67,8 +67,8 @@ find . -name "*.jpg" | wc -l
 ```yaml
 # Machine-readable Hub limits
 hub_limits:
-  max_file_size_gb: 50 # absolute hard stop enforced by LFS
-  recommended_file_size_gb: 20 # best-practice shard size
+  max_file_size_gb: 200 # absolute hard stop enforced by LFS
```

Comment:
is it LFS?

Reply:
Will open separate PR to replace `LFS` with `Large Files`